src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model/Migrations/20240416122016_InitialMigration.cs

```diff
@@ -579,6 +605,9 @@ protected override void Down(MigrationBuilder migrationBuilder)
             migrationBuilder.DropTable(
                 name: "ChangeSetTimePeriods");
 
+            migrationBuilder.DropTable(
```

Comment:
Are we going to continue to maintain the `Down()` methods in migrations, even though we never end up using them, do you think?

Reply:
I don't think we need to maintain them especially if involves spending time to write them in cases where we've had to write our own `Up` method from scratch.

I don't think there's any harm in including the autogenerated code though, like we see here.

I guess there could be confusion if the 'Down' method was ever relied upon on for something and the 'Up' and 'Down' methods hadn't been kept in sync. That could easily be the case right now where we've manually adjusted the content of the 'Up' method.

I would always try to make any alterations code first in the EF configuration so that the migration is autogenerated first, before resorting to writing any custom migration.